lib/common.js

```diff
@@ -162,7 +162,7 @@ function validateNftId(nftId) {
     if (isHex(nftId)) return new BN(nftId.substring(2), 16).toString(10);
     else {
       const hexNftId = new BN(nftId).toString(16);
-      if (hexNftId.length <= 64) return nftId;
+      if (nftId && hexNftId.length <= 64) return nftId;
       else throw nftId;
```

Comment:
So, if `nftId` is undefined we throw an undefined error? Shouldn't we have something else in here to explain that error?

Reply:
We have a catch clause that returns this message:
`catch (nftId) {
    throw new Error(`Invalid nftId type: ${nftId}`);
}`